scipy/special/add_newdocs.py

```diff
@@ -627,12 +672,40 @@ def add_newdoc(place, name, doc):
     E : ndarray
         Value of the elliptic integral.
 
+    Notes
+    -----
+    Wrapper for the Cephes [1]_ routine `ellpe`.
+
+    For `m > 0` the computation uses the approximation,
+
+    .. math:: E(m) \\approx P(1-m) - (1-m) \\log(1-m) Q(1-m),
```

Comment:
Minor comment for future: if you make the docstrings raw, you can use single backslashes too: , `r"""\frac{\epsilon}{\Theta^2}"""`. This way, TeX formulas look slightly more natural.